app/articles_download.py

```diff
@@ -0,0 +1,75 @@
+import requests
+import xmltodict
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session
+
+from app.config import FEED_URL
+from app.model import Article, Questions, Answers
+
+
+def download_data_from_url():
+    try:
+        response = requests.get(FEED_URL)
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as err:
+        raise SystemExit(err)
+    data = xmltodict.parse(response.content)
+    return data
+
+
+def update_articles(session):
+    articles = download_data_from_url()
+
+    counter = 0
+    for key, value in articles.items():
+        for node, article_data in value.items():
+            for q in article_data:
+                exist = session.query(Article).filter_by(article_id=q['ID']).all()
+                if not exist:
+                    counter = counter +1
+                    new_article = Article(article_id=q['ID'], published_date=q['CREATED_DATE'], title=q['TITLE'],
+                                          creator=q['AUTHOR'], image_src=q['IMAGE'], link_src=q['LINK'],
+                                          text=q['PEREX'],
+                                          keywords='TODO', media_name='cti-doma')
+                    session.add(new_article)
+                    session.commit()
+                    session.refresh(new_article)
+
+                    new_question = Questions(news_id=new_article.id, question_text=q['QUIZ']['QUIZ_TITLE'],
+                                             question_type=1, order=1)
+                    session.add(new_question)
+                    session.commit()
+                    session.refresh(new_question)
+
+                    order = 1 # tODO
+                    for key, answer in q['QUIZ']['QUIZ_OPTIONS'].items():
+                        for option in answer:
+                            new_answer = Answers(question_id=new_question.id,
+                                                 answer_text=option['OPTION_LABEL'],
+                                                 correct_answer_text= "TBD",
+                                                 correct_answers=True if option['CORRECT'] is not None else False,
+                                                 order=1)
+                            session.add(new_answer)
+                            session.commit()
+    print(f'Added {counter} articles.')
+
+
+def begin_import(event, context):
```

Comment:
`event` a `context` se nikde nepoužívají, ne?

Reply:
Je to tam kvůli AWSku, který potřebuje u funkce tyto dva parametry pokud se to má spustit jako cron lambda.